pixels/endpoints/general.py

```diff
@@ -0,0 +1,175 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException, Request, Response
+
+from pixels.constants import Ratelimits, Sizes
+from pixels.models import GetSize, Message, Pixel
+from pixels.utils import auth, ratelimits
+
+log = logging.getLogger(__name__)
+router = APIRouter(tags=["Canvas Endpoints"], dependencies=[Depends(auth.JWTBearer())])
+
+
+@router.get("/size", response_model=GetSize)
+async def size(request: Request) -> GetSize:
+    """
+    Get the size of the pixels canvas.
+
+    You can use the data this endpoint returns to build some cool scripts
+    that can start the ducky uprising on the canvas!
+
+    This endpoint doesn't require any authentication so dont worry
+    about giving any headers.
+
+    #### Example Python Script
+    ```py
+    import requests
+
+    r = requests.get("https://pixels.pythondiscord.com/get_size")
+    payload = r.json()
+
+    canvas_height = payload["height"]
+    canvas_width = payload["width"]
+
+    print(f"We got our canvas size! Height: {canvas_height}, Width: {canvas_width}.")
+    ```
+    """
+    return GetSize(width=Sizes.WIDTH, height=Sizes.HEIGHT)
+
+
+@router.get("/pixels", response_class=Response, responses={
+    200: {
+        "description": "Successful Response.",
+        "content": {
+            "application/octet-stream": {
+                "schema": {
+                    "type": "application/octet-stream",
+                    "format": "binary"
+                }
+            }
+        }
+    }
+})
+@ratelimits.UserRedis(
+    requests=Ratelimits.GET_PIXELS_AMOUNT,
+    time_unit=Ratelimits.GET_PIXELS_RATE_LIMIT,
+    cooldown=Ratelimits.GET_PIXELS_RATE_COOLDOWN
+)
+async def pixels(request: Request) -> Response:
+    """
+    Get the current state of all pixels from the database.
```

Comment:
Above you say "Get the size of the pixels canvas." so it might be best to refer to this as a canvas as well.
```suggestion
    Get the current state of all pixels from the canvas.
```